docs/topics/maven.md

```diff
@@ -23,6 +23,26 @@ To use JDK 17, in your `.mvn/jvm.config` file, add:
 --add-opens=java.base/java.io=ALL-UNNAMED
 ```
 
+## Declare repositories
+
+To access artifacts in repositories that can be consumed by Maven builds, specify the ID and URL of each repository in
+the `<repositories>` element:
+
+```xml
+<repositories>
+    <repository>
+        <id>mavenCentral</id>
+        <url>https://repo1.maven.org/maven2/</url>
+    </repository>
+</repositories>
+```
+
+> If you declare `mavenLocal()` as a repository and you also work with Gradle projects, then you
```

Comment:
It sounds like Maven has  `mavenLocal()` shortcut,but it is not. I think here we should say explicitly that 'mavenLocal() in Gradle build'.